pages/addRecipe.js

```diff
@@ -1,16 +1,35 @@
 import RecipeForm from "@/components/RecipeForm";
 import { useRouter } from "next/router";
 import useSWR from "swr";
+import React, { useState } from "react";
 
 export default function AddRecipe() {
+  const [imageUrl, setImageUrl] = useState("");
   const { mutate } = useSWR("/api/recipes");
   const router = useRouter();
 
+  const uploadImage = async (e) => {
+    const files = e.target.files;
```

Comment:
try to be explicit with the names of your parameters:

```suggestion
  const uploadImage = async (event) => {
    const files = event.target.files;
```